### ISSUE TITLE
Transfer account feature from one phone to another doesn't work

### ISSUE THREAD

**Issue by encrus** [mollyim/mollyim-android#245]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

Unable to transfer account as described in the title.

The error message I get is the following

"Error connecting"
An unexpected error occurred while attempting to connect to you new Android device

VERSIONS INFO:
Old phone :
Molly version : v5.50.4-1
Android version : 9

New phone :
Molly version : v6.40.4-1-FOSS
Android version : 13

SETUP INFO: 
Both phones on the same LAN
AFwall permissions set to for LAN, WiFi, Mobile Data

I am able to see the S8 phone in the WiFi Direct screen of the S23. I also got request for accept on S8 and accepted it

I have location, Wifi, location services etc all turned on on both devices

But I see "searching for new Android device" on S8 circling before retry option comes and 
"Waiting for old Android device to connect..." on S23 

Note that on the S8 phone, it said device no longer registered (which is of course understandable, I assume)

### Steps to reproduce

1.
2.
Pls see above

### Molly version

v6.40.4-1-FOSS

### Android version

Android 13

### Device

Samsung Galaxy S23

**Comment by encrus** [comment]
After lot of trials, I enabled/disabled WiFi on both phones (as suggested on the new molly / new phone screen). 
Then I tried to see each other devices under WiFi Direct - by going to its menu on both phones. I saw them both. 

Then it started working.
But after the transfer, Molly on S23 CRASHED ! 
I had to redo the process ! 
And again the same errors show up.
I think enabling and disabling WiFi seems to help. 

Also upon successful transfer on the new device - there is no notification or message. 
On the old device it does show, but on the new device (once it crashed and everything had to be redone, another time it just went to the next step - so wasn't clear if transfer was successful).
